tests/test_parsers.py

```diff
@@ -117,7 +117,7 @@ def test_parse_obographs(self):
             write_table(msdf, file)
         self.assertEqual(
             len(msdf.df),
-            9880,
+            9879,
```

Comment:
Why this change?

Reply:
Maybe something in the jsonld changed? It does a `wget`. I didn't dig deep to find out what happened.

Reply:
It's small enough a change to ignore